python/cudf/cudf/tests/test_indexing.py

```diff
@@ -2067,3 +2067,16 @@ def test_loc_index_notinindex_slice(
             expect = pdf.loc[lo:hi:take_order]
             actual = df.loc[lo:hi:take_order]
             assert_eq(expect, actual)
+
```

Comment:
If pandas upgrades are an intentional thing, then we can encode the pandas behaviour, and hence the TODO, in a test:

```
@pytest.mark.xfail(reason="https://github.com/pandas-dev/pandas/issues/54668")
def test_searchsorted_mixed_str_int():
    s = pd.Series([1, 2, 3], dtype="int")
    c = cudf.from_pandas(s)
    with pytest.raises(ValueError):
        actual = s.searchsorted("a")
    with pytest.raises(ValueError):
        expect = c.searchsorted("a")
    assert_eq(expect, actual)
```

This test will fail once that pandas issue is addressed, and remind us (when upgrading pandas) to change the cudf behaviour.

Reply:
Done 👍